AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -38,8 +38,11 @@ struct SessionHeaderView: View {
                     DeleteView(viewModel: DefaultDeleteSessionViewModel(session: session, measurementStreamStorage: measurementStreamStorage, streamRemover: DefaultSessionUpdateService(authorization: authorization, urlProvider: urlProvider), sessionSynchronizer: sessionSynchronizer), deleteModal: $showDeleteModal)
                 }
                 .sheet(isPresented: $showShareModal) {
-                    ShareSessionView(viewModel: DefaultShareSessionViewModel(session: session, exitRoute: {
+                    ShareSessionView(viewModel: DefaultShareSessionViewModel(session: session, apiClient: ShareSessionApi(urlProvider: urlProvider), exitRoute: { sharedEmail in
                         showShareModal.toggle()
+                        if sharedEmail == true {
```

Comment:
I don't quite like `sharedEmail` for a `bool` param here. How about changing it to `success` as usual?